x-pack/plugins/ml/public/application/components/data_grid/data_grid.tsx

```diff
@@ -188,6 +188,43 @@ export const DataGrid: FC<Props> = memo(
       );
     }
 
+    let errorCallout;
+
+    if (status === INDEX_STATUS.ERROR) {
```

Comment:
I'm assuming `errorMessage` will not be undefined here if status is equal to `INDEX_STATUS.ERROR`, but maybe it's safer to add a check here.

Reply:
Agree - better to explicitly check it. Added in 08b88797de85c4faabcc17f9b7d2488ab4b375ae.